modules/administration-guide/partials/con_architecture-overview-with-devworkspace.adoc

```diff
@@ -0,0 +1,27 @@
+[id="{prod-id-short}-architecture-overview-with-devworkspace{context}"]
+= {prod-short} architecture with {devworkspace}
+
+include::example$snip_using-devworkspace-warning.adoc[]
+
+.High-level {prod-short} architecture with the {devworkspace} engine
+image::architecture/che-interacting-with-devworkspace.png[]
+
+When {prod-short} is running with the {devworkspace} engine, it runs on three groups of components. The {prod-short} server components and the {devworkspace} engine are the two central server-side components of {prod-short}. Together they manage the User workspaces, which are container-based development environments.
```

Comment:
You say that "it runs on three groups of components" but the next sentence mentions only two server-side components. What's the third component? Consider adding a summary list of the 3 components before describing the 2 server-side components. Maybe something like this (revise for accuracy about the 3rd component):

`When {prod-short} is running with the {devworkspace} engine, it runs on three groups of components: two central server-side components and [client-side?] User workspaces.`

Reply:
The third one: user workspaces.